### PR TITLE
More in game mapping, moves maintenance generator to something that can be mapped in game

### PR DESCRIPTION
**Pull request by Bod9001**
NOTE:  Valley hasn't been done waiting for funtimedudeski  responsem if they have changes on the map
### Changelog:
CL: [New] Added system for custom gizmos for selected objects in VV
CL: [New] you can now select tiles in VV For a variable
CL: [Improvement] you can map maintenance generators now

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
image

myreactionsadtoinfo

**Comment by Bod9001** [comment]
it's just adding forever IDs to tiles

**Review by MaxIsJoe — DISMISSED** [review]
Nothing glaring in this PR's code.
I'd ask for you to reduce nesting and clean up the empty script, but the rest of the PR looks good enough for me to let it pass.

Merge once tests pass, unless Gilles has some input on it.

File: UnityProject/Assets/Scripts/Systems/MaintRooms/ExclusionZoneMono.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
+using System.Collections;
+using System.Collections.Generic;
+using Mirror;
+using Shared.Systems.ObjectConnection;
+using Systems.Scenes;
+using UnityEngine;
+
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/MaintRooms/ExclusionZoneMono.cs:8]: add namespace

**Review by corp-0 — CHANGES_REQUESTED** [review]
add namespaces to your new classes:
* ISOTracker 
* SOTracker 
* TileTracker 
* ExclusionZoneMono 
* ItemMatrixSystemInit 
* ISelectionGizmo
*

**Pull request merged**